test/test.ts

```diff
@@ -1,3 +1,6 @@
+/// <reference path="../node_modules/@types/mocha/index.d.ts" />
```

Comment:
The purpose of these is to ensure that within any environment that the types will be viewable for things within a test scope